db_store/src/lib.rs

```diff
@@ -31,6 +31,21 @@ impl<T> MetaValue<T> {
     }
 }
 
+macro_rules! query_exec_timed {
```

Comment:
shouldn't this macro be in the `poc-metrics` support crate so we could time other db access than just the db_store uses? the db_store is a shared behavior support crate for check-pointing pipeline progress within a particular service but then each service does _tons_ of db access "directly" via sqlx

Reply:
maybe? it just didn't seem well-thought-out-enough for general use.